scripts/rewards/verify_rewards.py

```diff
@@ -0,0 +1,72 @@
+from brownie.network.gas.strategies import GasNowStrategy
+from helpers.constants import DEFAULT_ADMIN_ROLE, PAUSER_ROLE, ROOT_PROPOSER_ROLE, ROOT_VALIDATOR_ROLE, UNPAUSER_ROLE
+from helpers.gnosis_safe import GnosisSafe, MultisigTxMetadata
+import json
+from assistant.rewards.rewards_checker import push_rewards, test_claims, verify_rewards
+from scripts.rewards.rewards_utils import calc_next_cycle_range
+import time
+
+from brownie import *
+from config.badger_config import badger_config
+from rich.console import Console
+from scripts.systems.badger_system import connect_badger
+
+from assistant.rewards.rewards_assistant import fetch_current_rewards_tree, fetch_pending_rewards_tree, run_action
+
+console = Console()
+
+gas_strategy = GasNowStrategy("rapid")
+
+def test_main():
+    main()
+
+def main():
+    badger = connect_badger(badger_config.prod_json, load_keeper=True, load_guardian=True)
+    pendingContentHash = "0x07baafa95bf7c39ba948753dda64b2e82854b5a7597f6de9e15c23a96d0bfad3"
```

Comment:
How was this hash computed?

Reply:
The hashes are computed from the merkle root of the pending  cycle . The current system works by encoding all the information about the claims in a [merkle tree](https://github.com/Darruma/badger-system/blob/master/assistant/rewards/merkle_tree.py). In this case the hash is hardcoded for testing purposes.